lib/cloudregister/smt.py

```diff
@@ -179,22 +193,27 @@ def __is_cert_valid(self, cert):
     # --------------------------------------------------------------------
     def __request_cert(self):
         """Request the cert from the SMT server and return the request"""
-        cert_rq = None
+        cert_res = None
         attempts = 0
         retries = 3
         while attempts < retries:
             attempts += 1
-            try:
-                ip = self.get_ipv4()
-                if not ip:
-                    ip = self.get_ipv6()
-                cert_rq = requests.get('http://%s/smt.crt' % ip)
-            except Exception:
-                # No response from server
-                logging.error('=' * 20)
-                logging.error('Attempt %s of %s' % (attempts, retries))
-                logging.error('Server %s is unreachable' % ip)
-            if cert_rq and cert_rq.status_code == 200:
-                attempts = retries
-
-        return cert_rq
+            for cert_name in ('smt.crt', 'rmt.crt'):
+                try:
+                    ip = self.get_ipv4()
+                    if not ip:
+                        ip = self.get_ipv6()
+                    cert_res = requests.get('http://%s/%s' % (ip, cert_name))
+                except Exception:
+                    # No response from server
+                    logging.error('=' * 20)
+                    logging.error(
+                        'Attempt %s with %s of %s' % (
+                            attempts, cert_name, retries)
+                    )
+                    logging.error('Server %s is unreachable' % ip)
+                if cert_res and cert_res.status_code == 200:
+                    attempts = retries
+                    break
```

Comment:
`break` should be enough to break out of the loop, `attempts = retries` shouldn't be needed anymore.